internal/mode/static/nginx/config/http/config.go

```diff
@@ -13,6 +13,8 @@ type Server struct {
 // Location holds all configuration for an HTTP location.
 type Location struct {
 	Return          *Return
+	PathRewrites    []string
+	HostnameRewrite string
```

Comment:
Will nginx expand `$` variables wrapped in double quotes? Because right now ProxySetHeaders is for user-provided headers and the value is quoted, versus our hard-coded headers that use variable expansion and aren't quoted.

Reply:
yep, `proxy_set_header host "$gw_api_compliant_host"` will work